java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/AbstractReader.java

```diff
@@ -57,19 +46,15 @@
     /** Contains the name of the plugin */
     protected final String pluginName;
 
-    /** The reader name (must be unique) */
-    protected final String name;
-
     /**
      * This flag is used with transmit or transmitSet
      * <p>
-     * It will be used by the notifySeProcessed method to determine if a request to close the
-     * physical channel has been already made and therefore to switch directly to the removal
-     * sequence for the observed readers.
+     * It will be used by the notifySeProcessed method (AbstractObservableLocalReader) to determine
+     * if a request to close the physical channel has been already made and therefore to switch
+     * directly to the removal sequence for the observed readers.<br>
+     * TODO find a better way to manage this need
      */
-    private boolean forceClosing = true;
-
-    /* ==== Constructor =================================================== */
+    protected boolean forceClosing = true;
```

Comment:
Is it possible to define it with a package access?

Reply:
No, because it is used in AbstractObservableLocalReader (not in the same package)